daemon/mgr/container.go

```diff
@@ -2224,6 +2236,104 @@ func (mgr *ContainerManager) populateVolumes(ctx context.Context, c *Container)
 	return nil
 }
 
+func (mgr *ContainerManager) setMountTab(ctx context.Context, c *Container) error {
```

Comment:
`mgr/container.go` are too big,  some functions are worked for container object, maybe we can put the code into `container_type.go`, WDYT???

Reply:
I add a new file `container_storage.go` to manager the operation of container storage module.